elementary/monitor/api/alerts/normalized_alert.py

```diff
@@ -58,7 +59,10 @@ def _flatten_meta(self, node_meta_field: str) -> dict:
     def _normalize_alert(self):
         try:
             normalized_alert = copy.deepcopy(self.alert)
-            normalized_alert[SUBSCRIBERS_KEY] = self._get_alert_subscribers()
+            normalized_alert[SUBSCRIBERS_KEY] = self._get_alert_meta_attrs(
+                SUBSCRIBERS_KEY
+            )
+            normalized_alert[OWNERS_KEY] = self._get_alert_meta_attrs("owner")
```

Comment:
You should use self.alerts.owners instead of extracting it from the metas.
Plus moving the email extraction from data monitoring alert to here seems like a good idea 🙂